src/actions/index.js

```diff
@@ -56,6 +60,21 @@ export const addProject = (project: Project) => ({
   project,
 });
 
+export const showDeleteProjectPrompt = (project: Project) => ({
+  type: SHOW_DELETE_PROJECT_PROMPT,
+  project,
+});
+
+export const deleteProjectFromDisk = (project: string) => ({
+  type: DELETE_PROJECT_FROM_DISK,
+  project,
+});
+
+export const deleteProjectFromDiskFinish = (deletedProject: string) => ({
+  type: FINISH_DELETING_PROJECT_FROM_DISK,
+  deletedProject,
```

Comment:
Couple nits about this variable:

- I think we should include the least amount of info with an action, and in this case, I think we could get away with just the `projectId`, rather than the whole project.
- I like when names of data entities are consistent, so I'd name this `project` instead of `deletedProject` (or `projectId` if we can just use the ID)